app/tests/spec/models/user.js

```diff
@@ -190,11 +191,12 @@ define(function (require, exports, module) {
 
     it('clearSignedInAccount', function () {
       sinon.spy(notifier, 'triggerRemote');
-      return user.setSignedInAccount({ email: 'email', uid: 'uid' })
+      const uid = createUid();
+      return user.setSignedInAccount({ email: 'email', uid })
```

Comment:
> It is hard for me to tell which/why it requires validation.

The uid is validated by the notifier [whenever it sends the SIGNED_OUT notification](https://github.com/mozilla/fxa-content-server/blob/39b8267dba27de5032c4ed79ddeb63d19e0f4f1c/app/scripts/lib/channels/notifier.js#L60).

> Should you make all user objects use valid emails?

User object probably not, but Account object probably. I'm 😨 (fearful) of the size of that change though. ;) We can try it and see what it busts.